responsibleai/responsibleai/managers/explainer_manager.py

```diff
@@ -97,6 +98,22 @@ def add(self):
             raise UserConfigValidationException(
                 'Model is required for model explanations')
 
+        if len(set(_find_features_having_missing_values(
+            self._initialization_examples)) &
+                set(self._categorical_features)) > 0:
+            raise UserConfigValidationException(
+                "Categorical features cannot have missing "
+                "values for computing explanations. "
+                "Please check your training data.")
+
+        if len(set(_find_features_having_missing_values(
+            self._evaluation_examples)) &
+                set(self._categorical_features)) > 0:
+            raise UserConfigValidationException(
```

Comment:
Alternatively, we could tell users which features have missing values. As a user, this sort of thing is very annoying to me since the library already did the work and still chose not to tell me the answer 😆 Same applies to all those occurrences, of course.

[In that case, you would not be able to use `any`, of course, since you actually need to say which features were found...]

Reply:
done